tst/EnergyPlus/unit/InputProcessor.unit.cc

```diff
@@ -261,8 +261,7 @@ TEST_F(InputProcessorFixture, decode_encode_2)
                                           "  Counterclockwise,",
                                           "  Relative,",
                                           "  Relative,",
-                                          "  Relative;"
-                                          "",
+                                          "  Relative;",
```

Comment:
This unit test doesn't seem to cover the proposed fix here.  I'll modify a test file to verify the fix, but should we also create a unit test for it here?

Reply:
It doesn't. I just cleaned up this unit test.

I tried to think of a way to unit test this but wasn't successful in my fairly quick attempt to fix this. The problem is that google test doesn't seem to have timeout functionality so how do you test for something that causes infinite looping?